### DISCUSSION TITLE
Should I use FastAPI to build an API to handle databases with millions of records

### DISCUSSION THREAD

**Discussion by MZaidAI** [fastapi/fastapi#3884] [Questions] [answered]
I am new to frameworks like Django-Rest-Framework, FastAPI, and Flask. I want to know if FastAPI will be an Ideal choice to build an API that can handle databases with millions of records. And is FastAPI scalable?

**Comment by ghandic** [comment, marked as answer]
Yes, FastAPI is scalable and can handle millions of records with the right deployment setup. You should look at best practices for the production deployment of REST API's. Check out the guide in the docs for more info. Let me know if you have any further questions.

https://fastapi.tiangolo.com/deployment/docker/

**Reply by MZaidAI** [reply]
Thank You So Much

**Reply by ghandic** [reply]
Could you mark as answered to close of and help others find the solution? :)

**Reply by MZaidAI** [reply]
> 
> 
> Could you mark as answered to close of and help others find the solution? :)

Done, sorry I had forgotten.
